StantDrawing/BrushTool/GroupSizeView.swift

```diff
@@ -0,0 +1,287 @@
+        buttonSize3.uiButton.frame.size = CGSize(width: 23, height: 23)
+        buttonSize4.uiButton.frame.size = CGSize(width: 28, height: 28)
+        buttonSize5.uiButton.frame.size = CGSize(width: 33, height: 33)
+        buttonSize6.uiButton.frame.size = CGSize(width: 37, height: 37)
+        
+        groupSizeButtons.append(buttonSize1)
+        groupSizeButtons.append(buttonSize2)
+        groupSizeButtons.append(buttonSize3)
+        groupSizeButtons.append(buttonSize4)
+        groupSizeButtons.append(buttonSize5)
+        groupSizeButtons.append(buttonSize6)
+    }
+    
+    private func configureGroupSizeLayout(groupButtons: [Button], groupSizeView: UIView, iconReferenceName: String, heightBaseToCenter: CGFloat, xBaseToCenter: CGFloat) {
+        // Icon
+        let iconView = createReferenceIconSizeView(iconReferenceName: iconReferenceName)
+        
+        // Separator
+        let yReference    = iconView.frame.origin.y + iconView.frame.height + 10
+        let separatorView = createSeparatorView(yReference: yReference)
+        
+        // Group Buttons
+        let yButtonsReference = separatorView.frame.origin.y + separatorView.frame.height + 10
+        let groupButtonsView  = configGroupButtonsView(groupButtons: groupButtons, yReference: yButtonsReference)
+        
+        // GroupSizeView Configs
+        let height    = iconView.frame.size.height + separatorView.frame.size.height + groupButtonsView.frame.size.height
+        let sizeGroup = CGSize(width: self.frame.width, height: height)
+        
+        groupSizeView.frame.size = sizeGroup
+        
+        groupSizeView.addSubview(iconView)
+        groupSizeView.addSubview(separatorView)
+        groupSizeView.addSubview(groupButtonsView)
+        
+        centrilizeVerticalView(view: groupSizeView, heightBaseToCenter: heightBaseToCenter, xBaseToCenter: xBaseToCenter)
+    }
+    
+    private func configGroupButtonsView(groupButtons: [Button], yReference: CGFloat) -> UIView {
+        let groupButtonsView = UIView()
+        let origin           = CGPoint(x: 0, y: yReference)
+        
+        groupButtonsView.frame.origin = origin
+        
+        configureGroupLayout(buttons: groupButtons, groupView: groupButtonsView, groupViewOirigin: origin, addAsSubview: false)
+        
+        setButtonsAsNotClicked(buttons: groupButtons)
+        
+        // Set the second SizeButton as Default
+        setButtonSizeAsClicked(button: groupButtons[1].uiButton)
+        
+        return groupButtonsView
+    }
+    
+    private func setButtonsAsNotClicked(buttons: [Button]) {
+        buttons.forEach { button in
+            setButtonSizeAsNotClicked(button: button.uiButton)
+        }
+    }
+    private func setButtonSizeAsClicked(button: UIButton) {
+        let darkGray = UIColor(hex: "606060")
+        button.alpha = 1
+        button.layer.cornerRadius = button.frame.height / 2
+        button.backgroundColor = darkGray
+    }
+    
```

Comment:
Indentation on these functions (setButtonSizeAsClicked and setButtonSizeAsNotClicked)